### PR TITLE
Create miscellaneous-functions.md

### PR DESCRIPTION
**Pull request by lilin90**
PTAL.

### CODE REVIEW

File: sql/miscellaneous-functions.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: 其他函数
+category: user guide
+---
+
+# 其他函数
+
+| 函数名 | 功能描述  |
+|:------|:-----------|
+| [`ANY_VALUE()`](https://dev.mysql.com/doc/refman/5.7/en/miscellaneous-functions.html#function_any-value) | 阻止 `ONLY_FULL_GROUP_BY` 值被拒  |
```
Comment (jackysp) [inline sql/miscellaneous-functions.md]: Is it a translation of the English version document?

Comment (jackysp) [reply sql/miscellaneous-functions.md]: 在`ONLY_FULL_GROUP_BY`模式下，防止带有`GROUP BY`的语句报错

File: sql/miscellaneous-functions.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+| [`SLEEP()`](https://dev.mysql.com/doc/refman/5.7/en/miscellaneous-functions.html#function_sleep)  | 休眠指定秒数   |
+| [`UUID()`](https://dev.mysql.com/doc/refman/5.7/en/miscellaneous-functions.html#function_uuid) | 返回通用唯一识别码 (UUID)   |
+| [`VALUES()`](https://dev.mysql.com/doc/refman/5.7/en/miscellaneous-functions.html#function_values)    | 定义 `INSERT` 过程中要用到的值  |
+| [`INET_ATON()`](https://dev.mysql.com/doc/refman/5.7/en/miscellaneous-functions.html#function_inet-aton)  | 返回 IP 地址的数值   |
```
Comment (jackysp) [inline sql/miscellaneous-functions.md]: 将 IP 地址转换为数值

File: sql/miscellaneous-functions.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+| [`INET_NTOA()`](https://dev.mysql.com/doc/refman/5.7/en/miscellaneous-functions.html#function_inet-ntoa)  | 从数值返回 IP 地址   |
```
Comment (jackysp) [inline sql/miscellaneous-functions.md]: 将数值转换为 IP 地址

File: sql/miscellaneous-functions.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+| [`INET6_ATON()`](https://dev.mysql.com/doc/refman/5.7/en/miscellaneous-functions.html#function_inet6-aton)  | 返回 IPv6 地址的数值    |
+| [`INET6_NTOA()`](https://dev.mysql.com/doc/refman/5.7/en/miscellaneous-functions.html#function_inet6-ntoa)  | 从数值返回 IPv6 地址  |
+| [`IS_IPV4()`](https://dev.mysql.com/doc/refman/5.7/en/miscellaneous-functions.html#function_is-ipv4)   | 判断参数是否为 IPv4 地址   |
+| [`IS_IPV4_COMPAT()`](https://dev.mysql.com/doc/refman/5.7/en/miscellaneous-functions.html#function_is-ipv4-compat)    | 判断参数是否为兼容 IPv4 的地址   |
+| [`IS_IPV4_MAPPED()`](https://dev.mysql.com/doc/refman/5.7/en/miscellaneous-functions.html#function_is-ipv4-mapped)    | 判断参数是否为 IPv4 映射的地址   |
+| [`IS_IPV6()`](https://dev.mysql.com/doc/refman/5.7/en/miscellaneous-functions.html#function_is-ipv6)    | 判断参数是否为 IPv6 地址    |
+| [`GET_LOCK()`](https://dev.mysql.com/doc/refman/5.7/en/miscellaneous-functions.html#function_get-lock)    | 获取命名锁      |
```
Comment (jackysp) [inline sql/miscellaneous-functions.md]: This function is not worked as its description. Actually it did nothing in TiDB

File: sql/miscellaneous-functions.md (hunk 1 above)
Comment (lilin90) [reply sql/miscellaneous-functions.md]: Yes, a translated version.
Got it. Thanks!

File: sql/miscellaneous-functions.md (hunk 4 above)
Comment (zz-jason) [reply sql/miscellaneous-functions.md]: @lilin90 plz address comments

**Comment by andelf** [comment]
LGTM

**Pull request merged**